app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -160,6 +165,22 @@ private String getSessionToken() {
         }
         return session[0];
     }
+    
+    void verifySupportedVersionOrThrowException(Study study, UserSession session) throws UnsupportedVersionException {
+
+    	// Exit early if Admin. These accounts should not throw the exception b/c of version support
+    	// b/c might want to look at older data. The supported version is intended for client apps that
+    	// gather data rather than apps used to model it. syoung 11/19/2015
+    	if (session.isAdminRole()) {
+    		return;  
+    	}
```

Comment:
I don't think we want to do this. These are orthogonal considerations and when we've made exceptions in the past for admin accounts, they've been used as participant accounts by developers and weird errors result.